internal/pgadmin/users.go

```diff
@@ -72,11 +74,47 @@ if sys.path[0] != root:
 	//
 	// TODO(cbandy): pgAdmin v4.21 adds "auth_source" and "username" as required attributes.
 	// - https://git.postgresql.org/gitweb/?p=pgadmin4.git;f=web/pgadmin/model/__init__.py;hb=REL-4_21#l65
+
+	// When the users are created or modified, server groups and connections will
+	// also be configured, similar to the way server groups and connections are
+	// made with their respective dialog windows.
+	// - https://www.pgadmin.org/docs/pgadmin4/latest/server_group_dialog.html
+	// - https://www.pgadmin.org/docs/pgadmin4/latest/server_dialog.html
+
+	// We use a similar method to the import method when creating server connections
+	// - https://www.pgadmin.org/docs/pgadmin4/development/import_export_servers.html
+	// - https://git.postgresql.org/gitweb/?p=pgadmin4.git;a=blob;f=web/setup.py;h=eb7d9044521a65781e6bdfebf02c8b2572f1dc97;hb=REL-4_20#l256
```

Comment:
⛏️ These links work for me without the `a=` and `h=` parameters. Do they work for others that way? If so, I'd prefer that we keep them out to keep the links shorter and maybe more permanent?

Same for the other links to git.postgresql.org.

Reply:
Yep, those work fine for me without those parameter. I'll make that change.